templates/advancement/item-choice-config.hbs

```diff
@@ -0,0 +1,104 @@
+<form autocomplete="off">
+  <div class="left-column">
+    {{> "dnd5e.advancement-controls"}}
+    <div class="form-group">
+      <label>{{localize "DND5E.AdvancementHint"}}</label>
+      <textarea name="configuration.hint">{{configuration.hint}}</textarea>
```

Comment:
I wouldn't object to making it a ProseMirror editor, but would there be any issue with sanitizing HTML data if we can't define this as an HTML field in `template.json`?

Reply:
That is a good point. This will be coming in v11 (https://github.com/foundryvtt/foundryvtt/issues/8680), but not for systems initially.

Even if we use `contenteditable` (assuming you can't copy & paste an `<iframe>` into a `contenteditable` `div`), we'd have to render it as HTML, meaning a user could update that field directly with malicious HTML.

I'll settle for some styling tweaks to the `<textarea>` for now, let me know what you think.

Reply:
<img width="359" alt="Screenshot 2023-01-16 at 13 17 36" src="https://user-images.githubusercontent.com/19979839/212767604-5114b553-a486-4a7c-ad4e-3cc432aa4fea.png">

What do you think of this? I've added a slight border back to match the other input elements. I also pushed the label up, which I feel flows a bit better, but I'm not set on it.

Reply:
Looks good to me